rwandaemr-installer/src/main/resources/scripts/vars.txt

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+## Distribution can be rwandaemr-imb-butaro or rwandaemr-imb-rwinkwavu
+distribution="rwandaemr-imb-butaro"
+## Version can either be 1.x or 2.x
+version="1.1.0-SNAPSHOT"
+## Any directory that you'd like your environment to be installed in
+target_dir="butaro"
+## env type can be demo, test, prod or dev
+env_type="dev"
+
+## Do not change these variable
+parent_dir="${target_dir}-${env_type}-${version}"
+username=${target_dir}${env_type}
```

Comment:
This is where I think you need to add more top-level logic.

Reply:
Not sure I follow